L1Trigger/L1TMuonEndCap/src/SectorProcessorShower.cc

```diff
@@ -8,12 +8,10 @@ void SectorProcessorShower::configure(const edm::ParameterSet& pset, int endcap,
   endcap_ = endcap;
   sector_ = sector;
 
+  enableOneLooseShower_ = pset.getParameter<bool>("enableOneLooseShower");
   enableTwoLooseShowers_ = pset.getParameter<bool>("enableTwoLooseShowers");
   enableOneNominalShower_ = pset.getParameter<bool>("enableOneNominalShowers");
   enableOneTightShower_ = pset.getParameter<bool>("enableOneTightShowers");
```

Comment:
I think these are the wrong name for these things now? Should be `Shower`?

Reply:
yes, exactly. Apparently I missed these. I changed them now.